setup.py

```diff
@@ -53,6 +54,36 @@ def read_version(package):
 ]
 
 
+class DockerUpCommand(Command):
+    description = "Start up docker compose with 3 bubuku and 1 zookeeper instances"
+    user_options = [
+        ('bubuku-scale=', None, 'Specify number of bubuku instances')
+    ]
+
+    def initialize_options(self):
+        self.bubuku_scale = 2
```

Comment:
Is this correct? I never used docker compose scale, but I was expecting it to be `3` given the description.